### PR TITLE
Proper error handling in AuthorizeController

### PR DESCRIPTION
**Pull request by Hussar77**
Changes proposed in this pull request:  
- Proper error handling in AuthorizeController
- 
- 

@ALM-Rangers/Team-Services-Bot

### CODE REVIEW

File: src/Team-Services-Bot.Api/Web.config [hunk 1]
```diff
@@ -14,7 +14,7 @@
   http://go.microsoft.com/fwlink/?LinkId=301879
   -->
 <configuration>
-  <appSettings configSource="AppSettings.config" />
+  <appSettings configSource="AppSettings.config.user" />
```
Comment (jeffrey-opdam) [inline src/Team-Services-Bot.Api/Web.config]: can you remove .user

Comment (Hussar77) [inline src/Team-Services-Bot.Api/Web.config]: That is intentional. I strongly believe that hacking of source control is really dangerous. Having AppAettings.config.user allows you to have your local settings for debugging and that file wouldn't go to source control.

Comment (jeffrey-opdam) [inline src/Team-Services-Bot.Api/Web.config]: Issue is that during runtime a user (contributor or viewer ) will miss the .user file. Which will stop them immediately. By adding the .user file, git will track the changes and we might by mistake commit it to the repo. 

I already tried these things with the appsettings.config file.

So could you please change it to AppSettings.config.

**Comment by Hussar77** [comment]
Corrected the AppSettings configuration file name.

**Pull request merged**